coffee-module/coffee-module-mp/coffee-module-mp-restclient/src/main/java/hu/icellmobilsoft/coffee/module/mp/restclient/exception/FaultTypeParser.java

```diff
@@ -46,15 +50,25 @@ public FaultTypeParser() {
      *            {@link String} to parse.
      * @return {@link hu.icellmobilsoft.coffee.dto.error.IFaultType} parsed from input string.
      */
+    @SuppressWarnings({ "unchecked", "rawtypes" })
     public static Enum<?> parseFaultType(String faultTypeString) {
-        for (Class<? extends Enum> faultTypeClass : FaultTypeParserExtension.getFaultTypeClasses()) {
-            Enum<?> fault = EnumUtils.getEnum(faultTypeClass, faultTypeString);
-            if (fault != null) {
-                return fault;
+        Instance<IFaultTypeProvider> faultProviderInstance = CDI.current().select(IFaultTypeProvider.class);
```

Comment:
ha valahol nem adnak meg implementaciot mi történik? nem keletkezik hiba?

Reply:
nem keletkezik. (arra van a feltetel benne, valamint ki is probaltam)